x-pack/plugins/maps/public/components/metrics_editor/metric_select.tsx

```diff
@@ -52,6 +52,12 @@ const AGG_OPTIONS = [
     }),
     value: AGG_TYPE.UNIQUE_COUNT,
   },
+  {
```

Comment:
Can you add this in alphabetical order, or, sort the `AGG_OPTIONS` by label so that the option is presented in alphabetical order?

<img width="250" alt="Screen Shot 2020-12-10 at 8 39 50 AM" src="https://user-images.githubusercontent.com/373691/101793923-905d4f00-3ac3-11eb-9495-0a971a8782c1.png">